projects/distributed-tracing/src/shared/dashboard/widgets/waterfall/waterfall/waterfall-chart.service.ts

```diff
@@ -41,7 +41,8 @@ export class WaterfallChartService {
         $$spanName: {
           name: datum.name,
           serviceName: datum.serviceName,
-          protocolName: datum.protocolName
+          protocolName: datum.protocolName,
+          error: datum.tags?.error as boolean
```

Comment:
isn't the error state a top level attribute? should we be fetching it directly rather than through the tag map (which means that it would be populated potentially via enrichment, since there's more than one way to mark an error).

Reply:
I don't think it is a top level attribute. Whenever I console log the data, I did not see any error attribute but there is one in the tags.
May be I am missing something.
cc: @kotharironak

Reply:
The top level attributes need to be added to the request, too. See the various waterfall data source models (for example from trace-waterfall-data-source.model.ts:
```ts 
  protected readonly spanSpecifications: Specification[] = [
    this.specificationBuilder.attributeSpecificationForKey('displaySpanName'),
    this.specificationBuilder.attributeSpecificationForKey('duration'),
    this.specificationBuilder.attributeSpecificationForKey('endTime'),
    this.specificationBuilder.attributeSpecificationForKey('parentSpanId'),
    this.specificationBuilder.attributeSpecificationForKey('serviceName'),
    this.specificationBuilder.attributeSpecificationForKey('spanTags'),
    this.specificationBuilder.attributeSpecificationForKey('startTime'),
    this.specificationBuilder.attributeSpecificationForKey('type'),
    this.specificationBuilder.attributeSpecificationForKey('traceId')
  ];
  ```

Reply:
@itssharmasandeep we should be using `EVENT. errorCount > 0` to indicate error. This value of this attribute is based on tags that indicate error at span level.
https://github.com/hypertrace/config-bootstrapper/blob/main/config-bootstrapper/src/main/resources/configs/config-bootstrapper/attribute-service/hypertrace/hypertrace_span_attributes.conf#L62

Reply:
Yeah, I am using the same now. 
updated the PR.